fuse_viz/src/relative_pose_2d_stamped_constraint_visual.cpp

```diff
@@ -32,31 +32,28 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include <fuse_viz/conversions.h>
-#include <fuse_viz/mapped_covariance_visual.h>
-#include <fuse_viz/pose_2d_stamped_visual.h>
-#include <fuse_viz/relative_pose_2d_stamped_constraint_visual.h>
-
-#include <fuse_constraints/relative_pose_2d_stamped_constraint.hpp>
-#include <fuse_core/graph.hpp>
-#include <geometry_msgs/msg/pose_with_covariance.hpp>
-#include <rviz_rendering/objects/axes.hpp>
-#include <rviz_rendering/objects/billboard_line.hpp>
-#include <rviz_rendering/objects/movable_text.hpp>
-#include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>  // NOLINT(build/include_order)
-#include <tf2/utils.h>
-
 #include <OgreQuaternion.h>
 #include <OgreSceneManager.h>
 #include <OgreSceneNode.h>
-
-#include <rclcpp/clock.hpp>
-#include <rclcpp/logging.hpp>
+#include <tf2/utils.h>
 
 #include <algorithm>
 #include <memory>
 #include <string>
 
+#include <fuse_constraints/relative_pose_2d_stamped_constraint.hpp>
+#include <fuse_core/graph.hpp>
+#include <fuse_viz/conversions.hpp>
+#include <fuse_viz/mapped_covariance_visual.hpp>
+#include <fuse_viz/pose_2d_stamped_visual.hpp>
+#include <fuse_viz/relative_pose_2d_stamped_constraint_visual.hpp>
+#include <geometry_msgs/msg/pose_with_covariance.hpp>
+#include <rclcpp/clock.hpp>
+#include <rclcpp/logging.hpp>
+#include <rviz_rendering/objects/axes.hpp>
+#include <rviz_rendering/objects/billboard_line.hpp>
+#include <rviz_rendering/objects/movable_text.hpp>
+#include <tf2_geometry_msgs/tf2_geometry_msgs.hpp>  // NOLINT(build/include_order)
```

Comment:
Does this no longer need the workaround where this header is included before `tf2/utils.h`?

Reply:
Hmm, it doesn't show up here, but according to the [issue](https://github.com/ros2/geometry2/issues/242), the linking issue happens sporadically.

I'll use the workaround just in case.
https://github.com/locusrobotics/fuse/pull/306/commits/6dde14ca699e18ee0f39ab2c6b045bad0e6127c0

Reply:
I had to put the NOLINT in a really weird position though...
https://github.com/locusrobotics/fuse/compare/6dde14ca699e18ee0f39ab2c6b045bad0e6127c0..679eba94513271846f9f80fabfbf771ba4315cd0